src/utils/client.ts

```diff
@@ -23,10 +25,12 @@ export const useApiClient = () => {
   apiClient.interceptors.response.use(
     function (successRes) {
       const newAcessToken = get(successRes.headers, "x-access-token", "");
+      const signal = abortController.signal;
+
       if (newAcessToken) {
         setAccessToken(newAcessToken);
       }
-      return successRes;
+      return { ...successRes, signal };
```

Comment:
You can revert those changes for now becouse it doesnt work correctly yet. I have to set it first.